js/libs/core.js

```diff
@@ -354,6 +354,25 @@ Fliplet.FormBuilder = (function() {
           }
         }
 
+        if (this._componentName === 'flSlider') {
+          if (!this.step) {
+            data.step = 1;
+          }
+
+
+          if (!this.min) {
+            data.min = 0;
+          }
+
+          if (!this.max) {
+            data.max = 100;
+          }
+
+          if ((Number(data.max) - Number(data.min)) % Number(data.step) != 0) {
```

Comment:
@yogeshmsharma-xoriant Can you explain why this didn't get picked up by ESLint?

```suggestion
          if ((Number(data.max) - Number(data.min)) % Number(data.step) !== 0) {
```

Reply:
seems linter is not getting applied for fliplet-widget-form-builder\libs\*,*.

Reply:
@yogeshmsharma-xoriant Can you make sure it is from now on please

Reply:
sure @tonytlwu. I didn't do any search on why it is not working in lib\***.  Will avoid it